searx/search/checker/impl.py

```diff
@@ -263,7 +263,7 @@ def _check_infoboxes(self, infoboxes):
     def check_basic(self):
         if len(self.result_container.unresponsive_engines) > 0:
             for message in self.result_container.unresponsive_engines:
-                self._record_error(message[1] + ' ' + (message[2] or ''))
+                self._record_error(message.error_type + ' ' + (str(message.suspended) if message.suspended else ''))
```

Comment:
Just double checking - is it safe to switch from indexer to a field/property access for this object? If so, just resolve this comment.

Reply:
> .. for this object?

this object is a named tuple / if we already give names to the items of a tuple, we should also use them :)

https://github.com/searxng/searxng/blob/74f6dfb576e66e17d3f968b252dfeee0a20eaf28/searx/results.py#L151-L154